### ISSUE TITLE
ignore typing.Generic in E0241: duplicate bases 

### ISSUE THREAD

**Issue by cedroyer** [PyCQA/pylint#2717]
### Is your feature request related to a problem? Please describe

When I use `typing.Generic` in 2 levels of inheritance (I think it's a correct way to use Generic), I don't want to have a `E0241: duplicate bases` message.

### Describe the solution you'd like

Ignore `typing.Generic` for `E0241: duplicate bases` duplicate test.

### Additional context
Add any other context about the feature request here.

short exemple:
```python
from typing import Generic, TypeVar

Number = TypeVar("Number")

class IVector(Generic[Number]):
      pass

class SparceVector(Generic[Number], IVector[Number]):
     pass

 # get `E0241: duplicate bases`.
class IntVector(SparceVector[int]):
    pass
```

**Comment by cedroyer** [comment]
Hi, thank you for your answer. Sorry for the mistake with `TypeVar`, I corrected it.

here is my pylint version:
```
% pylint --version
pylint 2.2.2
astroid 2.1.0
Python 3.7.2 (default, Jan 10 2019, 23:51:51) 
[GCC 8.2.1 20181127]
```
and the disable part of my `.pylintrc`:
```
[MESSAGES CONTROL]
disable=bad-continuation,
        too-many-ancestors,
        unsubscriptable-object,
        ungrouped-imports,
        wrong-import-order, 
        missing-docstring
```

**Comment by PCManticore** [comment]
Thanks @cedroyer I can reproduce it now.
